var/spack/repos/builtin/packages/steps/package.py

```diff
@@ -13,7 +13,7 @@ class Steps(CMakePackage):
     git      = "git@github.com:CNS-OIST/HBP_STEPS.git"
 
     version("develop", branch="master", submodules=True)
-    version("3.5.0a",  commit="1c8a5b0", submodules=True)
```

Comment:
Is this actually getting re-built? I recall commit hash changes not changing the hash of the package… :/

Reply:
Humm... indeed it seems that even though tests re-run, it didn't recompile Steps.
But is it problematic? only difference is it's rebuilding another commit, which has been tested

Reply:
Something to keep in mind. Unexpected effects, and why I'm not a fan of moving versions/commits around.